src/encoded/static/components/search.js

```diff
@@ -315,7 +315,9 @@ const ExperimentComponent = (props, reactContext) => {
             biosamples = _.compact(result.replicates.map(replicate => replicate.library && replicate.library.biosample));
         }
         // flatten treatment array of arrays
-        _.compact(biosamples.map(biosample => biosample.treatments)).forEach(treatment => treatment.forEach(t => treatments.push(t)));
+        _.compact(biosamples.map(biosample => biosample.treatments)).forEach(treatment => treatment.forEach((t) => {
```

Comment:
foreach in a foreach is a code smell; unless you are dealing with something like tree and hate recursion. Are you trying to do something like:

`[...[1,2,3], ...[4,5,6]]`

or something like reduce(https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce)